.github/workflows/test.yml

```diff
@@ -119,7 +89,7 @@ jobs:
         sudo apt update
         sudo apt -y install criu
 
-    - name: install CRIU (criu ${{ matrix.criu }})
+    - name: install CRIU (${{ matrix.criu }})
```

Comment:
How about remove this one? And using `else` here?

Reply:
Yes, I think we can use `else` in `bash`, or else we will see a log like this in CI:
```
Install CRIU ()
```